### PR TITLE
Added machine-config to change MTU

### PR DESCRIPTION
**Pull request by manuvaldi**
Added machine-config to change MTU in a second interface

### CODE REVIEW

File: content/machine-config/_index.md [hunk 1]
```diff
@@ -133,3 +133,52 @@ spec:
         path: /etc/sysctl.d/99-elasticsearch.conf
 EOF
 ```
+
+# Modify MTU in a second interface in workers
+
+Sometimes a storage network interface is attached to nodes in order to reach an external storage. Probably you could need to modify the MTU in those interfaces. 
+
+You can do that adding an an script in dispatcher.d for NetworkManager service in the path `/etc/NetworkManager/dispatcher.d/`. But if SELinux is enabled in your installation you could have errors when the daemon execute that script. For that you should add a new one-shot systemd service for modify the context.
```
Comment (e-minguez) [inline content/machine-config/_index.md]: `an an`

File: content/machine-config/_index.md [hunk 2]
```diff
@@ -133,3 +133,52 @@ spec:
         path: /etc/sysctl.d/99-elasticsearch.conf
 EOF
 ```
+
+# Modify MTU in a second interface in workers
+
+Sometimes a storage network interface is attached to nodes in order to reach an external storage. Probably you could need to modify the MTU in those interfaces.
```
Comment (e-minguez) [inline content/machine-config/_index.md]: `Sometimes a storage network interface is attached to nodes in order to reach an external storage. In order to improve the performance, you could need to modify the MTU in those interfaces to 9000 (aka. jumboframes)`

File: content/machine-config/_index.md [hunk 3]
```diff
@@ -133,3 +133,52 @@ spec:
...
 EOF
 ```
+
+# Modify MTU in a second interface in workers
+
+Sometimes a storage network interface is attached to nodes in order to reach an external storage. Probably you could need to modify the MTU in those interfaces. 
+
+You can do that adding an an script in dispatcher.d for NetworkManager service in the path `/etc/NetworkManager/dispatcher.d/`. But if SELinux is enabled in your installation you could have errors when the daemon execute that script. For that you should add a new one-shot systemd service for modify the context. 
+
+In this example the MTU of the interface `ens4` will change to `9000` (jumbo)
```
Comment (e-minguez) [inline content/machine-config/_index.md]: In this example the MTU of the `ens4` interface will change to `9000` to enable jumboframes:

File: content/machine-config/_index.md [hunk 4]
```diff
@@ -133,3 +133,52 @@ spec:
...
+
+```
+cat << EOF | oc create -f -
+kind: MachineConfig
+apiVersion: machineconfiguration.openshift.io/v1
+metadata:
+  name: 99-worker-mtu-v2
```
Comment (e-minguez) [inline content/machine-config/_index.md]: `name: 99-worker-mtu`

File: content/machine-config/_index.md [hunk 5]
```diff
@@ -133,3 +133,52 @@ spec:
...
+  osImageURL: ''
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - filesystem: root
+        path: "/etc/NetworkManager/dispatcher.d/30-mtu"
+        contents:
+          source: data:,%23%21%2Fbin%2Fsh%0AMTU%3D9000%0AINTERFACE%3Dens4%0A%0AIFACE%3D%241%0ASTATUS%3D%242%0Aif%20%5B%20%22%24IFACE%22%20%3D%20%22%24INTERFACE%22%20-a%20%22%24STATUS%22%20%3D%20%22up%22%20%5D%3B%20then%0A%20%20%20%20ip%20link%20set%20%22%24IFACE%22%20mtu%20%24MTU%0Afi%0A
```
Comment (e-minguez) [inline content/machine-config/_index.md]: Would you mind to make this a bit more readable? :D You can paste the script before or after the machine config content so readers can see it at a glance.

File: content/machine-config/_index.md (hunk 1 above)
Comment (manuvaldi) [reply content/machine-config/_index.md]: Change done, thanks

File: content/machine-config/_index.md (hunk 2 above)
Comment (manuvaldi) [reply content/machine-config/_index.md]: Change done, thanks

File: content/machine-config/_index.md (hunk 3 above)
Comment (manuvaldi) [reply content/machine-config/_index.md]: Change done, thanks

File: content/machine-config/_index.md (hunk 4 above)
Comment (manuvaldi) [reply content/machine-config/_index.md]: Change done, thanks

File: content/machine-config/_index.md (hunk 5 above)
Comment (manuvaldi) [reply content/machine-config/_index.md]: Take a look to the new commit, please

**Pull request merged**